Assets/Scripts/LevelTransitions/Plane.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
```

Comment:
This file has wrong indentation, it mixes spaces and tabs. Please make it conform to the coding style.